src/lib.rs

```diff
@@ -1728,7 +1728,7 @@ impl<R: Read + Seek> MatroskaFile<R> {
                                     self.file.seek(SeekFrom::Start(offset + size))?;
                                 }
                                 _ => {
-                                    // We found the first element after the seeked timestamp.
+                                    // We found the first element after the sought timestamp.
```

Comment:
Is this really a fix? "seeked" might not be a classical english word, but it has been used a lot in codified work like for example:

https://html.spec.whatwg.org/multipage/media.html#event-media-seeked